examples/Huggingface_Transformers/Image_classification_docker/HF-only/README.md

```diff
@@ -0,0 +1,158 @@
+
+
+# HF only
+
+## Setup
+Log into the terminal of an AWS EC2 `t2.micro` instance with a base AWS Deep Learning PyTorch AMI (as it includes [`torch-model-archiver`](https://github.com/pytorch/serve/tree/master/model-archiver) and `docker`)
+
+The `Using Torchserve Docker containers on EC2 for inference` section of [this Notion blog](https://www.notion.so/Day-2-3-Torchserve-custom-handlers-and-Docker-containers-02665de910a64aedab2b907a9a0cc9b0#3825849dfb8942379df2cdce8a729d9a) by me can be followed for logging into the appropriate AWS EC2 instance.
+
+
+## Running the Torchserve server
+Within the AWS EC2 instance, activate the PyTorch environment and navigate to the current folder after cloning this repo: 
+
+```
+source activate pytorch
+git clone https://github.com/tripathiarpan20/HF-torchserve-pipeline
```

Comment:
let's keep the article self contained within this repo, what dependencies for that repo would you need to have here. It's easier to share maintenance burden in the future if everything is in the same place

Reply:
The line `source activate pytorch` is essentially only supposed to make these packages/libraries available in the Linux shell:
* torch-model-archiver
* git
* docker
* curl
* git-lfs (installed via yum in the README as it's not available in AWS EC2 PyTorch 1.12 AMI)

I suppose we could cut out the 'running on AWS EC2' part and include this 'running with docker container' part from the `Image_classification_docker` example, that way we don't have to deal with the ambiguity of the `pytorch` environment that is currently referred.

Meanwhile, I shall keep my repo centered around running the HF models with Torchserve on AWS EC2 instance & integration with Huggingface `optimum` library.